htdocs/footer.html

```diff
@@ -1,55 +1,33 @@
-	<li><a href="<!--#echo var="relroot" -->whatsnew.html" title="The changes made to this service recently">News</a><span> | </span></li>
-	<li><a href="<!--#echo var="relroot" -->docs/" accesskey="3" title="Documentation for this Service">Docs</a><span> | </span></li>
-	<li><a href="<!--#echo var="relroot" -->docs/help.html" title="Help and answers to frequently asked questions">Help&nbsp;&amp;&nbsp;<abbr title="Frequently Asked Questions">FAQ</abbr></a><span> | </span></li>
-	<li><a href="<!--#echo var="relroot" -->feedback.html" accesskey="4" title="How to provide feedback on this service">Feedback</a></li>
-	<li><a href="<!--#echo var="relroot" -->contribute.html" title="How to contribute to the validator project">Contribute</a></li>
-</ul>
+    <li><a href="/" accesskey="1" title="Go to the Home Page for The W3C Markup Validation Service">Home</a><span> | </span></li>
+    <li><a href="/about.html" title="Information About this Service">About...</a><span> | </span></li>
 
-<div id="footer">
-	<p id="activity_logos">
-		<a href="http://www.w3.org/Status" title="W3C's Open Source, bringing you free Web quality tools and more">
-		<img src="//www.w3.org/Icons/WWW/w3c_home_nb" alt="W3C" width="72" height="47" />
-		<img src="<!--#echo var="relroot" -->images/opensource-55x48.png" alt="Open-Source" title="We are building certified Open Source/Free Software. - see www.opensource.org" width="55" height="48" /></a>
-	</p>
+    <li><a href="/whatsnew.html" title="The changes made to this service recently">News</a><span> | </span></li>
+    <li><a href="/docs/" accesskey="3" title="Documentation for this Service">Docs</a><span> | </span></li>
+    <li><a href="/docs/help.html" title="Help and answers to frequently asked questions">Help&nbsp;&amp;&nbsp;<abbr title="Frequently Asked Questions">FAQ</abbr></a><span> | </span></li>
+    <li><a href="/feedback.html" accesskey="4" title="How to provide feedback on this service">Feedback</a><span> | </span></li>
+    <li><a href="/contribute.html" title="How to contribute to the validator project">Contribute</a><span> | </span></li>
+  </ul>
+  <div id="footer">
 
-	<p id="support_logo">
-		<a href="http://www.w3.org/QA/Tools/Donate">
-			<img src="//www.w3.org/QA/Tools/I_heart_validator" alt="I heart Validator logo" title=" Validators Donation Program" width="80" height="15" />
-		</a>
-	</p>
-	<p id="version_info">
-		This service runs the W3C Markup Validator, <a href="<!--#echo var="relroot" -->whatsnew.html#v14"><abbr title="version">v</abbr>1.3+hg</a>.
-	</p>
+    <p id="activity_logos">
+    <a href="https://status.w3.org/" title="W3C's Open Source, bringing you free Web quality tools and more"><img src="https://www.w3.org/assets/logos/w3c/w3c-no-bars.svg" alt="W3C" width="72" height="47"><img src="./images/opensource-55x48.png" alt="Open-Source" title="We are building certified Open Source/Free Software. - see www.opensource.org" width="55" height="48"></a>
```

Comment:
Link to the W3C Open Source software page (and not to W3C Systems Status page).

```suggestion
    <a href="https://www.w3.org/Status" title="W3C's Open Source, bringing you free Web quality tools and more"><img src="https://www.w3.org/assets/logos/w3c/w3c-no-bars.svg" alt="W3C" width="72" height="47"><img src="./images/opensource-55x48.png" alt="Open-Source" title="We are building certified Open Source/Free Software. - see www.opensource.org" width="55" height="48"></a>
```